account_invoice_show_currency_rate/models/account_move.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    currency_rate_amount = fields.Float(
+        string="Rate amount", compute="_compute_currency_rate_amount", digits=0,
+    )
+    show_currency_rate_amount = fields.Boolean(
+        compute="_compute_show_currency_rate_amount", readonly=True
+    )
+
+    @api.depends(
+        "state",
+        "date",
+        "line_ids.amount_currency",
+        "company_id",
+        "currency_id",
+        "show_currency_rate_amount",
+    )
+    def _compute_currency_rate_amount(self):
+        """ It's necessary to define value according to some cases:
+        - Case A: Currency is equal to company currency (Value = 1)
+        - Case B: Move exist previously (posted) and get real rate according to lines
+        - Case C: Get expected rate (according to date) to show some value in creation.
+        """
+        for item in self:
+            item.currency_rate_amount = 1
```

Comment:
You can assign in one shot:
```suggestion
        self.currency_rate_amount = 1
```